images/linux/scripts/installers/cleanup.sh

```diff
@@ -27,4 +27,15 @@ after=$(df / -Pm | awk 'NR==2{print $4}')
 # display size
  echo "Before: $before MB"
  echo "After : $after MB"
- echo "Delta : $(($after-$before)) MB"
\ No newline at end of file
+ echo "Delta : $(($after-$before)) MB"
+
+ # Remove quotes if found around PATH
+if [[ $PATH == \"*\" ]]
+then
+    PATH=${PATH#"\""}
+    PATH=${PATH%"\""}
+    echo "PATH=$PATH" | sudo tee -a /etc/environment
+    echo "Updated path to remove quotes: $PATH"
+else
+    echo "No path update needed: $PATH"
+fi
```

Comment:
Do we really need this hack now?
It looks like a hotfix to fix some issue with PATH. Can we check if it is still actual?

Reply:
hack removed :)